using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -953,7 +1174,41 @@ Before using this API, see [Identify unique visitors](https://docs.adobe.com/con
 This API gets a custom Analytics visitor identifier, which has been set previously using [setVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#setidentifier). Callback with NSError if an unexpected error occurs or the request times out.
 
 {% tabs %}
-{% tab title="iOS" %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### getVisitorIdentifier
+
+#### Syntax
+
+```swift
+static func getVisitorIdentifier(completion: @escaping (String?, Error?) -> Void)
+```
+
+* _callback_ is invoked with the visitor identifier value.
+
+#### Example
+
+**Swift**
+
+```swift
+Analytics.getVisitorIdentifier { (visitorId, error) in
+   // check the visitorIdentifier value
+}
+```
+
+**Objective-C**
+
+```objectivec
+[AEPMobileAnalytics getVisitorIdentifier:^(NSString * _Nullable visitorIdentifier, NSError * _Nullable error) {
+    // check the visitorIdentifier value or handle error
+}];
+```
+
+{% endtab %}
```

Comment:
Same here, let's not duplicate this code and reference the getVisitorIdentifier section instead.